code/modules/mob/living/simple_animal/hostile/giant_spider.dm

```diff
@@ -50,6 +50,9 @@
 	pass_flags = PASSTABLE
 	move_to_delay = 6
 	ventcrawler = 2
+	nestedProc = "Web"

```

Comment:
The verbs it calls have "set name" in them, causing it to runtime because it tries to call typepath/"full name" instead of typepath/actualname. I'm going to use strings instead.